pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -1035,6 +1035,37 @@ cdef class _Timedelta(timedelta):
 
     @property
     def seconds(self) -> int:  # TODO(cython3): make cdef property
+        """
+        Return timedelta in number of seconds converting hours and minutes in seconds.
```

Comment:
Maybe `Return the total hours, minutes, and seconds of the timedelta as seconds`

To make it clear that no other component is included.

Reply:
Done